ui/src/institutions/components/InstitutionsHistoricalDataList.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import InlineList, {
+  SEPARATOR_MIDDLEDOT,
+} from '../../common/components/InlineList';
+
+function renderHistoricalNote(historicalNote) {
+  return <span>{historicalNote}</span>;
+}
+
+function extractKeyFromData(historicalNote) {
+  return historicalNote;
+}
+
+function InstitutionsHistoricalDataList({ historicalData }) {
+  return (
+    <InlineList
+      label="Note"
+      items={historicalData}
+      extractKey={extractKeyFromData}
+      renderItem={renderHistoricalNote}
```

Comment:
you don't have to set `extractKey` and `renderItem` for this case, because `InlineList` has defaults.